src/Raven.Server/Smuggler/Documents/StreamSource.cs

```diff
@@ -1758,9 +1783,133 @@ private async IAsyncEnumerable<DocumentItem> ReadDocumentsAsync(List<string> col
                         }
                     }
 
+                    // hashBySize == null means there were no attachment streams in the dump
+
+                    //TODO: egor do this only if the dump is old (pre v6.2) (LessThenCurrent)
```

Comment:
??

Reply:
added a if statement on `_buildVersionType` but need to edit it  according to version that the feature will be released in